docs/apps/rbac/for_clients.md

```diff
@@ -0,0 +1,114 @@
+## Using DAB RBAC as an API Client
+
+This section tells how to use the API endpoints as a client;
+what requests to make and how to build request data.
+
+You need a server running a Django project that uses this system.
+Use test_app in this repo for a demo, see `test_app/README.md` for bootstrap.
+http://127.0.0.1:8000/admin/ allows login as admin user (password is "password").
+
```

Comment:
Yeah, this is a valid issue. I think it would be best to file it in https://github.com/ansible/django-ansible-base/issues, because in that case it's unclear what to change (the demo data command, or the script?), so the issue is a good place to formalize the decision.

Reply:
I'll go ahead and open a new issue.

Reply:
#471